lib/tapioca/dsl/compilers/active_record_relations.rb

```diff
@@ -821,24 +818,21 @@ def create_common_methods
                 return_type: "T.any(Integer, Float, BigDecimal)",
               )
             when :count
-              sigs = [
-                common_relation_methods_module.create_sig(
-                  parameters: { column_name: "T.nilable(T.any(String, Symbol))" },
-                  return_type: "Integer",
-                ),
-                common_relation_methods_module.create_sig(
-                  parameters: { column_name: "NilClass", block: "T.proc.params(object: #{constant_name}).void" },
-                  return_type: "Integer",
-                ),
-              ]
-              common_relation_methods_module.create_method_with_sigs(
-                "count",
-                sigs: sigs,
-                parameters: [
-                  RBI::OptParam.new("column_name", "nil"),
-                  RBI::BlockParam.new("block"),
-                ],
-              )
+              common_relation_methods_module.create_method(method_name.to_s) do |method|
```

Comment:
Yes, it was, and I actually wanted to do it for other methods as well, but Copilot made many of the changes for me, so I lost steam.

The reason why I want to use `method_name.to_s` is because it is better for future extension. If there is another method that behaves exactly as `count` (let's say they add an alias for `count`), then it will be enough to add that to the `where` clause, since the method name is dynamic like this.